event/error.go

```diff
@@ -0,0 +1,18 @@
+package event
```

Comment:
Do you we can use `dp-api-clients-go/errors`? (or, as this is not an api call, we may decide to duplicate it for now?)